src/util/createMetaModule.js

```diff
@@ -40,7 +35,13 @@ module.exports = function createMetaModule(mapFunction, moduleOptions) {
     function draw(input, callback) {
 
       var step = this;
-      
+
+      // map inputs to internal step options;
+      // use this to set defaults for internal steps
+      // and to expose internal settings as external meta-module parameters;
+      // it must return a steps object
+      var steps = mapFunction(options);
```

Comment:
Hi @tech4GT can you check this change of scope? Thank you!

Reply:
Linking to explanation: https://github.com/publiclab/image-sequencer/issues/1239#issuecomment-571337844